kube/pod.go

```diff
@@ -68,8 +68,12 @@ func NewPodTemplate(role *model.Role, settings *ExportSettings) (helm.Node, erro
 	container.Add("readinessProbe", readinessProbe)
 	container.Sort()
 
+	imagePullSecrets := helm.NewMapping()
+	imagePullSecrets.Add("name", "registry-credentials")
```

Comment:
If a mapping has just a single entry then I would add that in the constructor call:

```
imagePullSecrets := helm.NewMapping("name", "registry-credentials")
```